sdks/python/apache_beam/runners/portability/expansion_service.py

```diff
@@ -63,6 +64,10 @@ def with_pipeline(component, pcoll_id=None):
       }
       transform = with_pipeline(
           ptransform.PTransform.from_runner_api(request.transform, context))
+      if request.output_coder_override.spec.urn:
+        output_coder = Coder.from_runner_api(
+            request.output_coder_override, context)
+        transform = transform.with_output_types(output_coder.to_type_hint())
```

Comment:
This could be unsafe. Should we at least make sure it's compatible? (This would have to happen after application...)